ticker/simpleMock.go

```diff
@@ -0,0 +1,58 @@
+package ticker
+
+import (
+	"sync/atomic"
+	"time"
+)
+
+// SimpleMock implements the Ticker interface, and provides a method of
+// force-feeding ticks, even while paused.
+type SimpleMock struct {
+	IsActive uint32 // used atomically
```

Comment:
If this is exposed publicly, then it allows callers to use the variable _non_ atomically. Instead, we should force them to access this variable _only_ via the methods defined below (if we actually need them).

Reply:
Good point, hidden behind method now.